.gitignore

```diff
@@ -1,125 +1,11 @@
-# Created by https://www.gitignore.io/api/java,gradle,kotlin,intellij
-# Edit at https://www.gitignore.io/?templates=java,gradle,kotlin,intellij
-
 ### Intellij ###
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and WebStorm
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
-# User-specific stuff
-.idea/**/workspace.xml
-.idea/**/tasks.xml
-.idea/**/usage.statistics.xml
-.idea/**/dictionaries
-.idea/**/shelf
-
-# Generated files
-.idea/**/contentModel.xml
-
-# Sensitive or high-churn files
-.idea/**/dataSources/
-.idea/**/dataSources.ids
-.idea/**/dataSources.local.xml
-.idea/**/sqlDataSources.xml
-.idea/**/dynamic.xml
-.idea/**/uiDesigner.xml
-.idea/**/dbnavigator.xml
-
-# Gradle
-.idea/**/gradle.xml
-.idea/**/libraries
-
-# Gradle and Maven with auto-import
-# When using Gradle or Maven with auto-import, you should exclude module files,
-# since they will be recreated, and may cause churn.  Uncomment if using
-# auto-import.
-# .idea/modules.xml
-# .idea/*.iml
-# .idea/modules
- *.iml
-# *.ipr
-
-# CMake
-cmake-build-*/
-
-# Mongo Explorer plugin
-.idea/**/mongoSettings.xml
-
-# File-based project format
-*.iws
+.idea/**
```

Comment:
My intention is only manually add ignore for those files we want.
Generated file is too big. It will be difficult to support it correct in the future

Reply:
Could you simply help me make our manual file more precise

Reply:
It's very easy to edit this. The link is at the top of the file:
```
# Created by https://www.gitignore.io/api/java,gradle,kotlin,intellij
# Edit at https://www.gitignore.io/?templates=java,gradle,kotlin,intellij
```

Problem is if you add `.idea/**` now then copyright settings will be removed. as well as default codestyle later